src/Components/App/App.js

```diff
@@ -127,7 +135,16 @@ class App extends Component {
                 this.state.isSearching
                 ) {
                   return <Redirect to="/searchResults" />;
-                } else {
+              } else if(
```

Comment:
I would like to create a helper function for this Route (is it possible to include helper functions on Router?).

Reply:
We can look into it today! @gaj23